pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -2011,6 +2011,7 @@
     introducing: "Introducing"
     music_lab: "Music Lab"
     introducing_music_lab: "Introducing Music Lab"
+    introducing_music_lab_strong: "Introducing **Music Lab**"
```

Comment:
Found a good css workaround to add the icon on the `<strong>` element that wraps `**Music Lab**`:
<img width="257" alt="Screenshot 2024-04-24 at 10 22 03 AM" src="https://github.com/code-dot-org/code-dot-org/assets/9256643/eb5028ac-5f6a-4600-b6e9-4039d76b1f48">

Reply:
Good question @mgc1194! I asked Karim and Michael to weigh in on [Slack](https://codedotorg.slack.com/archives/C04TH8400AU/p1714067901608119).

Reply:
Closing the loop: we are going to keep this as-is for now. See the Slack convo above for more context.

Reply:
Re-opening the loop 😅 The XT has decided they **do not** want "Music Lab" to be translated, so I updated this string to make that so.